public/assets/scss/_variables.scss

```diff
@@ -198,3 +198,28 @@ $input-plaintext-color: $body-color !default;
 $input-btn-focus-box-shadow: none;
 
 $input-group-addon-bg: transparent;
+
+// Popovers
```

Comment:
Let's remove this since you are not making use of the variables. If you update the variables, you don't need to add a custom popover class. I don't think you totally understand my last comment, You might need to read the previous comment again.